File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/InsetItemDecoration.java
package com.xwray.groupie.example.viewbinding;

import androidx.annotation.ColorInt;
import androidx.annotation.Dimension;

public class InsetItemDecoration extends com.xwray.groupie.example.core.decoration.InsetItemDecoration {
    public InsetItemDecoration(@ColorInt int backgroundColor, @Dimension int padding) {
        super(backgroundColor, padding, MainActivity.INSET_TYPE_KEY, MainActivity.INSET);
    }
}

File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/InsetItemDecoration.kt
package com.xwray.groupie.example.viewbinding

import androidx.annotation.ColorInt
import androidx.annotation.Dimension
import com.xwray.groupie.example.core.decoration.InsetItemDecoration

class InsetItemDecoration(
    @ColorInt backgroundColor: Int,
    @Dimension padding: Int
) : InsetItemDecoration(backgroundColor, padding, INSET_TYPE_KEY, INSET)
